src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -37,8 +37,8 @@ private sealed class BoundSpillSequenceBuilder : BoundExpression
             private ArrayBuilder<LocalSymbol> _locals;
             private ArrayBuilder<BoundStatement> _statements;
 
-            public BoundSpillSequenceBuilder(BoundExpression value = null)
-                : base(SpillSequenceBuilderKind, null, value?.Type)
+            public BoundSpillSequenceBuilder(SyntaxNode syntax, BoundExpression value = null)
```

Comment:
`nullable enable` and `SyntaxNode?`

Reply:
Not in this PR.

---
In reply to: [416896544](https://github.com/dotnet/roslyn/pull/43471#discussion_r416896544) [](ancestors = 416896544)